data-prepper-plugins/aggregate-processor/README.md

```diff
@@ -82,8 +82,8 @@ While not necessary, a great way to set up the Aggregate Processor [identificati
        * `count_key`: key name to use for storing the count, default name is `aggr._count`
        * `start_time_key`: key name to use for storing the start time, default name is `aggr._start_time`
        * `output_format`: format of the aggregated event.
-         * Default is OTEL SUM metric type with count as value
-         * Other supported output format is `raw` which generates JSON with `count_key` field with count as value and `start_time_key` field with aggregation start time as value
+         * `otel_metrics` - Default output format. Outputs in otel metrics format, etc.
```

Comment:
Haha I meant the `etc` to be a filler for the real description of this output format